### ISSUE TITLE
Different phrasing for same setting

### ISSUE THREAD

**Issue by Kefaku** [modrinth/knossos#1600]
### Describe the bug

The Visibility settings of a project will show the following wording if set to private, unlisted or archived:

```
If approved by the moderators:

Will be visible in search
Will be visible on profile
Will be visible via URL 
```

Only if the setting is set to listed, will the wording change to this:

```
If approved by the moderators:

Visible in search
Visible on profile
Visible via URL 
```

### Steps to reproduce

1. Go to the settings of any project.
2. Select the "General" section.
3. Scroll to "Visibility".
4. Change the setting.

### Expected behavior

The wording of the description/information should be the same, no matter what the setting is currently is.

### Additional context

![listed](https://github.com/modrinth/knossos/assets/114934849/35d8e6f8-c0be-4fb7-a9ff-14d64f16ead6)
---
![unlisted](https://github.com/modrinth/knossos/assets/114934849/8f805e23-c690-4c41-a030-b8aee02b00b4)

**Comment by Kefaku** [comment]
I just looked into the source code, changing that text seems to be somewhat intentional, but it still seems illogical to me, so I'm leaving this bug report open.

**Comment by triphora** [comment]
Yes, this is intentional. The "Visible" wording is used for whatever the setting is at that exact moment. The "Will be visible" is describing your pending change.

**Comment by Kefaku** [comment]
> The "Visible" wording is used for whatever the setting is at that exact moment.
@triphora 

But this "Visible" wording is also shown in the settings even when I change the settings on a project not yet submitted for review. In that case, "Visible in search" and "Visible in profile" can't possibly reflect the current state of the project, because it's not yet listed.

I know that might be an unimportant detail, I just want to understand it.
